packages/shared/src/components/notifications/NotificationIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import {
+  notificationDefaultTheme,
+  notificationIcon,
+  NotificationIcon,
+} from '../../graphql/notifications';
+
+const noBackgroundIcons = [
+  NotificationIcon.DailyDev,
+  NotificationIcon.CommunityPicks,
+];
+
+interface NotificationItemIconProps {
+  icon: NotificationIcon;
+}
+
+function NotificationItemIcon({
+  icon,
+}: NotificationItemIconProps): ReactElement {
+  const Icon = notificationIcon[icon] ?? notificationIcon.DailyDev;
```

Comment:
If you are defaulting to a value, why not use the default value above:
```
function NotificationItemIcon({
  icon = NotificationIcon.DailyDev,
}: NotificationItemIconProps): ReactElement {
```

Reply:
That is just the props, we are defaulting to the value given by the `notificationIcon` object.